### DISCUSSION TITLE
monthly view when replying on dates

### DISCUSSION THREAD

**Discussion by timokvamme** [lukevella/rallly#167] [Ideas]
Hi, thanks for the amazing software.

I love everything about it, its perfect.
Except for the view when deciding a date, where there is no option
to view it over a month, or week view. 

Currently the view is like so:
![image](https://user-images.githubusercontent.com/6280756/167577626-1e1df270-903a-4a9e-9bbb-09a9621c76dd.png)

It would suit my needs better if there was a monthly view, similar to when creating the schedule.

In case there  are multiple timeslots on a given date, then participants can click on individual dates, in a monthly view
and select from a list of options on that given day, rather than having to sidescroll until they find a date.
Moreover the monthly view is more standard and similar to how participants usually look at calendars.

![image](https://user-images.githubusercontent.com/6280756/167577813-e8ba674b-619f-4f83-8bf9-00361a6ad6d8.png)

Best Regards

**Comment by lukevella** [comment]
Hey @timokvamme thanks for the feedback and I agree that it would be really useful especially for polls with lots of options. It's not quite at the front of the priority list yet but maybe in a few weeks I'll be able to get to this.

**Comment by timokvamme** [comment]
Cheers for the reply.  I still plan to use Rallly despite the lack of this feature as I can't find a better free option than Rallly at the moment.

**Comment by joeauyeung** [comment]
I'm trying to plan a trip and one way I'm getting around this is by getting participants to choose weekends that do not work for them.

**Comment by matcasti** [comment]
Oh this would be so cool! And much better to search across multiple weeks appointments
